apiclient/harvester_api/managers/base.py

```diff
@@ -0,0 +1,89 @@
+import json
+from weakref import ref
+
+from pkg_resources import parse_version
+
+DEFAULT_NAMESPACE = "default"
+
+
+def merge_dict(src, dest):
+    for k, v in src.items():
+        if isinstance(dest.get(k), dict) and isinstance(v, dict):
+            merge_dict(v, dest[k])
+        else:
+            dest[k] = v
+    return dest
+
+
+class BaseManager:
+    _sub_classes = dict()
+    support_to = "0.0.0"
```

Comment:
Please help to add explanation comment on key properties/methods, helps grasp quickly.
* `support_to`
* `is_support`
* `for_version`

Reply:
add some docstring, but how it could be use is the flexibility (it should not be the doc of attributes/methods), let me add a comment to show how will it work.